react/react2/inclass/app/blogs/[blogId]/page.jsx

```diff
@@ -0,0 +1,26 @@
+"use client";
+import { usePathname, useSearchParams } from "next/navigation";
+
+export default function blog({ params }) {
+  const path = usePathname();
+  const query = useSearchParams();
+
+  function removeHyphens(str) {
+    const words = str.split("-");
+
+    // Capitalize the first letter of each word except the first one
+    const camelCased = words.map((word, index) => {
+      return word[0].toUpperCase() + word.slice(1);
+    });
+    return camelCased.join(" ");
+  }
```

Comment:
Just a performance note: this function going to be recreated every time react rerenders, but it is not necessary, because your function is not binded to this component.

So instead of
```
function Blog() {
...
const removeHyphens = () => {}
...
}
```
you can do
```
const removeHyphens = () => {}

function Blog() {
...
}
```